src/components/questionStructure/Task/variants/HandwritingTask/types.ts

```diff
@@ -0,0 +1,10 @@
+import { ExcalidrawElement } from '@excalidraw/excalidraw/types/element/types'
+import { AppState } from '@excalidraw/excalidraw/types/types'
+
+export interface HandwritingAnswer {
+  excalidraw: {
+    elements: readonly ExcalidrawElement[]
+    appState?: Omit<AppState, 'offsetTop' | 'offsetLeft' | 'width' | 'height'>
```

Comment:
Does the appState **need** saving as part of the answer?

Reply:
Our idea with saving appState was that if a user refreshed the page or went out and then back in again to the canvas the state of that canvas was saved. Namely, this includes where a user was on the canvas (zoom & position), which I think is good for quality-of-usage. It means if you were just editing something in the canvas you'll end up back at the thing you were editing to pick off writing from there.

Reply:
Of course, long-term we'd probably not store it in the answer field but e.g. in a separate table. But for now everything is just dumped into the answer field of the DB, so that's the only place we have to store stuff

Reply:
OK fair enough